backend/src/models/report.js

```diff
@@ -225,6 +226,40 @@ const getProjectStatuses = (projectId) => {
     .where("ps.project_id", projectId)
     .orderBy("ps.id", "DESC");
 };
+
+// Get array of deliverables with summed budget amounts.
+const getDeliverableBudgets = (projectId, fiscal, quarter) => {
+  return knex(`${projectBudgetTable} as pb`)
+    .select("pd.deliverable_name", {
+      amount: knex.raw(`SUM(pb.q${quarter}_amount::numeric::float8)`),
+    })
+    .join(`${projectDeliverableTable} as pd`, "pb.project_deliverable_id", "pd.id")
+    .where("pd.project_id", projectId)
+    .andWhere("pb.fiscal", fiscal)
+    .groupBy("pd.id")
+    .orderBy("pd.deliverable_name", "ASC")
+    .having(knex.raw(`SUM(pb.q${quarter}_amount::numeric::float8)`), ">", 0);
```

Comment:
this is interesting about the quarter.

Reply:
There might be an issue here if we don't validate properly. The frontend only allows values 1-4 but it's still possible to get a quarter 5 here which would cause the query to fail since there is no q5_amount column.